sqlite-migrations/src/main/kotlin/com/squareup/sqlite/migrations/CatalogDatabase.kt

```diff
@@ -0,0 +1,45 @@
+package com.squareup.sqlite.migrations
+
+import schemacrawler.schema.Catalog
+import schemacrawler.schemacrawler.SchemaCrawlerOptions
+import schemacrawler.schemacrawler.SchemaInfoLevelBuilder
+import schemacrawler.utility.SchemaCrawlerUtility
+import java.sql.Connection
+import java.sql.DriverManager
+import java.sql.SQLException
+
+class CatalogDatabase private constructor(
+  internal val catalog: Catalog
```

Comment:
in the future we might want to make `Catalog` and schemacrawler an implementation detail of this module, and instead pass it just `.db` files and `SqliteStatement`s. For now I think its good to just abstract stuff out, api will become clearer once IDEA plugin starts using this

Reply:
Yeah, at some point implementations can move into a separate module